src/Agent.Plugins/PipelineCache/SavePipelineCacheV0.cs

```diff
@@ -29,6 +29,18 @@ public class SavePipelineCacheV0 : PipelineCacheTaskPluginBase
             string salt,
             CancellationToken token)
         {
+            if(!context.Variables.ContainsKey("agent.jobstatus"))
```

Comment:
Nit: can we use `Constants.Variables.Agent.JobStatus` instead of the hard-coded string?

Reply:
Spoke with @TingluoHuang . We are yet to add Constants.cs in the sdk. His suggestion it to to "agent.jobstatus"

Reply:
Ugggh. OK :)